am-ui/webcomponents/create-ad-webcomponent/tests/unit/CreateAd.spec.js

```diff
@@ -0,0 +1,82 @@
+import CreateAd from '@/components/CreateAd.ce.vue';
+import {createI18n} from 'vue-i18n';
+import {shallowMount} from '@vue/test-utils';
+
+// jest.mock('./../../src/components/CreateAd.ce.vue', () => ({

```

Comment:
Is this still needed?

Reply:
Let's just keep it because if tests are to be improved at some point this should be the first thing to do, not best practices I know but doesn't hurt.